Source/Eto.WinForms/Forms/Controls/DropDownHandler.cs

```diff
@@ -158,18 +158,6 @@ public DropDownHandler()
 			};
 		}
 
-		public override Color BackgroundColor
-		{
-			get
-			{
-				return Control.BackColor.ToEto();
-			}
-			set
-			{
-				Control.BackColor = value.ToSD();
-			}
-		}
-
```

Comment:
This was the same as original base implementation. Removed to enable new transparent background on drawable (as `base.BackgroundColor` now manages `BackgroundColorSet`)